board/main.c

```diff
@@ -219,7 +219,7 @@ int usb_cb_control_msg(USB_Setup_TypeDef *setup, uint8_t *resp, int hardwired) {
       // so it's blocked over wifi
       switch (setup->b.wValue.w) {
         case 0:
-          if (hardwired) {
+          if (hardwired != 0) {
```

Comment:
Change hardwired to bool

Reply:
yes, it will be a bit of refactor (hardwired is an input param in few functions), but it will be cleaner.

Reply:
97a11115bbec51be0b6d56b3ac3f8461d64633b7